adapters/amx/amx.go

```diff
@@ -0,0 +1,174 @@
+package amx
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strings"
+)
+
+const vastImpressionFormat = "<Impression><![CDATA[%s]></Impression>"
+const vastSearchPoint = "</Impression>"
+
+// AmxAdapter is the AMX bid adapter
+type AmxAdapter struct {
+	endpoint string
+}
+
+// NewAmxBidder creates an AmxAdapter
+func NewAmxBidder(endpoint string) *AmxAdapter {
+	return &AmxAdapter{endpoint: endpoint}
+}
+
+type amxExt struct {
+	Bidder amxParams `json:"bidder"`
+}
+
+type amxParams struct {
+	PublisherID string `json:"tagId,omitempty"`
```

Comment:
Please move `amxParams` to the `openrtb_ext` package in the file openrtb_ext/imp_amx.go with the name `ExtImpAmx` to match our standard convention.